source/auditing-logging.txt

```diff
@@ -161,11 +164,68 @@ In addition to the following examples, see the blogpost
    .. tab:: CLI
       :tabid: cli
 
+      Update Audit Configuration
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      Run the following {+atlas-cli+} command to audit all authentication events
+      for known users in your project:
+
+      .. include:: /includes/examples/cli-example-audit-logs-known-users.rst
+
+      Run the following {+atlas-cli+} command to audit a known user
+      via a configuration file:
+
+      .. include:: /includes/examples/cli-example-audit-logs-config-file.rst
+
+      Describe Audit Configuration
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      Run the :ref:`atlas auditing describe <atlas-auditing-describe>`
+      {+atlas-cli+} command to return the auditing configuration for the specified project:
+
+      .. include:: /includes/examples/cli-example-audit-logs-describe.rst
+
+      Create and Use Audit Filter
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      Create an audit filter to only audit the authenticate operations
+      that occur against the test database. To learn more,
+      see :manual:`Audit Messages </reference/audit-message/>`.
```

Comment:
Can we instead link here? https://www.mongodb.com/docs/manual/tutorial/configure-audit-filters/